modules/030-cloud-provider-yandex/monitoring/prometheus-rules/deprecated_availability_zone.yaml

```
@@ -12,15 +12,21 @@
         plk_protocol_version: "1"
         plk_create_group_if_not_exists__cluster_has_node_groups_deprecation_alerts: "AvailabilityZonesDeprecationAlerts,tier=cluster,prometheus=deckhouse,kubernetes=~kubernetes"
         plk_grouped_by__cluster_has_node_groups_deprecation_alerts: "AvailabilityZonesDeprecationAlerts,tier=cluster,prometheus=deckhouse,kubernetes=~kubernetes"
-        summary: NodeGroup {{ $labels.node_group }} contains Nodes with deprecated availability zone.
+        summary: Node group `{{ $labels.node_group }}` contains nodes in a deprecated availability zone.
```

```suggestion
        summary: NodeGroup `{{ $labels.node_group }}` contains nodes in a deprecated availability zone.
```

@maxinsky the kind of resourse - NodeGroup